recipes/gstreamer/all/test_package/CMakeLists.txt

```diff
@@ -1,15 +1,28 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.6)
 project(test_package LANGUAGES C)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
 find_package(gstreamer REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME}
-    gstreamer::gstreamer-1.0 gstreamer::gstreamer-base-1.0 gstreamer::gstreamer-controller-1.0
-    gstreamer::gstreamer-net-1.0 gstreamer::gstreamer-check-1.0)
-if (TARGET gstreamer::gstcoreelements)
-    target_link_libraries(${PROJECT_NAME} gstreamer::gstcoreelements)
-endif ()
+
+if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
why does Windows need to have a different path?

Reply:
I copied this from the `glib` package: https://github.com/conan-io/conan-center-index/blob/master/recipes/glib/all/test_package/CMakeLists.txt#L9-L20

Reply:
@uilianries do you remember? pkg-config should work on Windows as well

Reply:
I think it's related to https://github.com/conan-io/conan/issues/11962 CMakeToolchain + pkg config is bugged.

Reply:
okay, good to know...
so no need to change it now.
I personally prefer to avoid unnecessary conditions in our test_packages, as they make code larger and harder to read and understand, and our test packages are often used as an examples of conan best practices (the fact you copied code from GLib only confirms that).

Reply:
@SSE4 it does not smell good, but we have few options. Anyway, having Conan client fixed will solve that workaround, but we will need to wait.